src/xcode/ENA/ENA/Source/SceneDelegate.swift

```diff
@@ -297,10 +297,11 @@ extension SceneDelegate: ENAExposureManagerObserver {
 }
 
 extension SceneDelegate: HomeViewControllerDelegate {
+	/// Resets all stores and notifies the Onboarding.
 	func homeViewControllerUserDidRequestReset(_: HomeViewController) {
-		store.isOnboarded = false
-		store.dateLastExposureDetection = nil
+		store.clearAll()
 		UIApplication.coronaWarnDelegate().downloadedPackagesStore.reset()
```

Comment:
good point, I forgot to tell you to remove downloadedPackagesStore.reset().

Reply:
Removed with https://github.com/corona-warn-app/cwa-app-ios/pull/168/commits/24d1f9fa749014f5f8d412af17e7ab508604d9cb

Reply:
Hi @llanghammer-sap ,
Sorry, I think we have misunderstood here. :-)
We need to remove the downloadPakcageStore.
Thanks.